### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
- Fixed container margin for manage dns component
- Moved retry button and status text out of gpopper slot

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user

```

### CODE REVIEW

File: frontend/src/components/ShootStatus.vue [hunk 1]
```diff
@@ -5,61 +5,63 @@ SPDX-License-Identifier: Apache-2.0
...
-              <v-icon v-else-if="isAborted && isShootLastOperationTypeDelete" class="vertical-align-middle cursor-pointer status-icon" :color="color">mdi-delete</v-icon>
-              <v-icon v-else-if="isAborted && isShootMarkedForDeletion" class="vertical-align-middle progress-icon" :color="color">mdi-delete-clock</v-icon>
-              <v-icon v-else-if="isAborted && isTypeCreate" class="vertical-align-middle cursor-pointer status-icon" :color="color">mdi-plus</v-icon>
-              <v-icon v-else-if="isUserError" class="vertical-align-middle cursor-pointer status-icon" :color="color">mdi-account-alert</v-icon>
-              <v-icon v-else-if="isError" class="vertical-align-middle cursor-pointer status-icon" :color="color">mdi-alert-outline</v-icon>
-              <v-progress-circular v-else-if="isPending" class="vertical-align-middle cursor-pointer" :size="27" :width="3" indeterminate :color="color"></v-progress-circular>
-              <v-icon v-else class="vertical-align-middle cursor-pointer status-icon-check" color="success">mdi-check-circle-outline</v-icon>
+  <div class="d-flex">
```
Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:8]:
```suggestion
  <div class="d-flex align-center">
```
```suggestion
  <div class="d-flex">
```

Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:8]:
```suggestion
  <div class="d-flex align-center">
```
```suggestion
  <div class="d-flex">
```

File: frontend/src/components/ShootStatus.vue [hunk 2]
```diff
@@ -5,61 +5,63 @@ SPDX-License-Identifier: Apache-2.0
...
+    <g-popper :title="statusTitle" :toolbar-color="color" :popper-key="popperKeyWithType" :placement="popperPlacement">
+      <template v-slot:popperRef>
+        <div class="shoot-status-div d-flex flex-row">
```
Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:11]: Remove this div. I think we don't need it anymore.

Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:11]: Remove this div. I think we don't need it anymore.

File: frontend/src/components/ShootStatus.vue [hunk 3]
```diff
@@ -5,61 +5,63 @@ SPDX-License-Identifier: Apache-2.0
...
+        :status-title="statusTitle"
+        :last-message="lastMessage"
+        :error-descriptions="errorDescriptions"
+        :last-update-time="shootLastOperation.lastUpdateTime"
+        :secret-binding-name="shootSecretBindingName"
+        :namespace="shootNamespace"
+      />
+    </g-popper>
+    <retry-operation :shoot-item="shootItem"></retry-operation>
+    <span v-if="showStatusText" class="d-flex align-center ml-2">{{statusTitle}}</span>
```
Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:63]:
```suggestion
    <span v-if="showStatusText" class="ml-2">{{statusTitle}}</span>
```
```suggestion
    <span v-if="showStatusText" class="d-flex align-center ml-2">{{statusTitle}}</span>
```

Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:63]:
```suggestion
    <span v-if="showStatusText" class="ml-2">{{statusTitle}}</span>
```
```suggestion
    <span v-if="showStatusText" class="d-flex align-center ml-2">{{statusTitle}}</span>
```

File: frontend/src/components/ShootStatus.vue (hunk 2 above)
Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:11]: Remove this div. I think we don't need it anymore.

Comment (holgerkoser) [reply frontend/src/components/ShootStatus.vue:11]: I am not sure if some styles of this css class are still needed but `display: inline-block` makes no sense in combination with `d-flex`
```css
.shoot-status-div {
    display: inline-block;
    width: 30px;
    text-align: center;
    height: auto;
    max-height:  30px;
  }
```

File: frontend/src/components/ShootStatus.vue [hunk 4]
```diff
@@ -216,7 +212,7 @@ export default {
 
   }
 
-  .progress-icon {
+  .pr on {
```
Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:215]: I think this is wrong?

File: frontend/src/components/ShootStatus.vue [hunk 5]
```diff
@@ -5,61 +5,65 @@ SPDX-License-Identifier: Apache-2.0
...
-    <shoot-message-details
-      :status-title="statusTitle"
-      :last-message="lastMessage"
-      :error-descriptions="errorDescriptions"
-      :last-update-time="shootLastOperation.lastUpdateTime"
-      :secret-binding-name="shootSecretBindingName"
-      :namespace="shootNamespace"
-    />
-  </g-popper>
+  </span>
```
Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:66]:
```suggestion
  </div>
```

File: frontend/src/components/ShootStatus.vue [hunk 6]
```diff
@@ -5,61 +5,65 @@ SPDX-License-Identifier: Apache-2.0
...
-            <span v-if="tooltip.progress" class="ml-1">({{tooltip.progress}}%)</span>
-            <div v-for="({ shortDescription, userError }) in tooltip.errorCodeObjects" :key="shortDescription">
-              <v-icon v-if="userError" class="mr-1" color="white" small>{{ userError ? 'mdi-account-alert' : 'mdi-alert' }}</v-icon>
-              <span class="font-weight-bold text--lighten-2">{{shortDescription}}</span>
-            </div>
-          </div>
-        </v-tooltip>
-        <retry-operation :shoot-item="shootItem"></retry-operation>
-        <span v-if="showStatusText" class="d-flex align-center ml-2">{{statusTitle}}</span>
+  <span>
```
Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:8]:
```suggestion
  <div>
```

File: frontend/src/components/ShootStatus.vue (hunk 4 above)
Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:215]:
```suggestion
  .progress-icon {
```

File: frontend/src/components/ShootStatus.vue [hunk 7]
```diff
@@ -5,61 +5,65 @@ SPDX-License-Identifier: Apache-2.0
...
+              <div>
+                <span class="font-weight-bold">{{tooltip.title}}</span>
+                <span v-if="tooltip.progress" class="ml-1">({{tooltip.progress}}%)</span>
+                <div v-for="({ shortDescription, userError }) in tooltip.errorCodeObjects" :key="shortDescription">
+                  <v-icon v-if="userError" class="mr-1" color="white" small>{{ userError ? 'mdi-account-alert' : 'mdi-alert' }}</v-icon>
+                  <span class="font-weight-bold text--lighten-2">{{shortDescription}}</span>
+                </div>
+              </div>
+            </v-tooltip>
+          </span>
```
Comment (holgerkoser) [inline frontend/src/components/ShootStatus.vue:46]:
```suggestion
          </div>
```

File: frontend/src/components/ShootStatus.vue (hunk 2 above)
Comment (grolu) [reply frontend/src/components/ShootStatus.vue]: I think we can remove the class

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**